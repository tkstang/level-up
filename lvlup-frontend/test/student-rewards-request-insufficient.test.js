import RequestInsufficient from '../src/components/Student/student-reward-request/reward-request-insufficient';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import thunk from 'redux-thunk';
import configureMockStore from 'redux-mock-store';
import lvlupApp from '../src/reducers/index';

const middlewares = [thunk];
const mockStore = configureMockStore(middlewares);

test('RequestInsufficient should render', () => {
  const store = mockStore({ lvlupApp });
  const req = shallow(
    <RequestInsufficient store={store} />,
  );
  expect(shallowToJson(req)).toMatchSnapshot();
});
