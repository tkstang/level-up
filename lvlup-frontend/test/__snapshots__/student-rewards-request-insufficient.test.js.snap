exports[`test RequestInsufficient should render 1`] = `
<Message
  color="orange"
  content="Unfortunately you do not have sufficient points for this reward...seems like a good time to lvl^"
  header="Insufficient Points!"
  onDismiss={[Function]} />
`;
